internal/app/singularity/remote_get_login_password.go

```diff
@@ -0,0 +1,55 @@
+package singularity
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	scslibclient "github.com/sylabs/scs-library-client/client"
+)
+
+func RemoteGetLoginPassword(config *scslibclient.Config) (string, error) {
+	client := http.DefaultClient
+	path := "/v1/rbac/users/current"
+	endPoint := config.BaseURL + path
+
+	req, err := http.NewRequest(http.MethodGet, endPoint, nil)
+	if err != nil {
+		return "", fmt.Errorf("error creating request: %v", err)
+	}
+	req.Header.Set("Authorization", fmt.Sprintf("Bearer %v", config.AuthToken))
+	res, err := client.Do(req)
+	if err != nil {
+		return "", fmt.Errorf("error making request to server: %v", err)
+	}
+	defer res.Body.Close()
+
+	if res.StatusCode != http.StatusOK {
+		return "", fmt.Errorf("unexpected response from server, status: %v", res.Status)
+	}
+
+	var u User
+	err = json.NewDecoder(res.Body).Decode(&u)
+	if err != nil {
+		return "", fmt.Errorf("unable to marshal response into json: %v\nfailed to get cli token", err)
+	}
+
+	if u.OidcUserMeta.Secret == "" {
+
+		return "", fmt.Errorf("failed to get cli token")
+	}
```

Comment:
one last thing, maybe make the UX message something like "this library does not support direct OCI access"